### PR TITLE
[TACHYON-1497] Improve tfs mkdir shell command to process multiple args

### PR DESCRIPTION
**Pull request by luoli523**
tfs shell command mkdir cannot create multiple dir in one command, like below:

    bin/tachyon tfs mkdir /dir1 /dir2

improve the command to support this convenience.

### CODE REVIEW

File: shell/src/main/java/tachyon/shell/command/MkdirCommand.java [hunk 1]
```diff
@@ -51,24 +51,35 @@ protected int getNumOfArgs() {
...
   public String getDescription() {
-    return "Creates the specified directory, including any parent directories that are required.";
+    return "Creates the specified directories, including any parent directories that are required.";
+  }
+
+  @Override
+  public boolean validateArgs(String... args) {
+    boolean valid = args.length >= getNumOfArgs();
+    if (!valid) {
+      System.out.println(getCommandName() + " takes " + getNumOfArgs() + " arguments at least\n");
```
Comment (yupeng9) [inline shell/src/main/java/tachyon/shell/command/MkdirCommand.java]: arguments -> argument

**Comment by yupeng9** [comment]
Thanks for the improvement. Could you add a unit test?

**Comment by yupeng9** [comment]
LGTM. Thanks for the improvement!

**Comment by yupeng9** [comment]
@luoli523 could you update the command description in the docs 'Command Line Interface'?

**Comment by luoli523** [comment]
update the mkdir d command line description in 'Command Line Interface' doc

**Comment by luoli523** [comment]
Seems like this failure has nothing to do with this patch?
I think it's a issue already recorded here?
https://tachyon.atlassian.net/browse/TACHYON-1534

**Comment by yupeng9** [comment]
Jenkins, test this please

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**